cclient/api/core/core_init.h

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Can we just call this `core.h` ?
And maybe merge it with `core_api.h` as we will end up including both anyway ?

Reply:
The user should include `core_api.h` for core API only and `extended_api.h` for both core and extended, see `ccliet/api/BUILD`. 
`core_init.h` is included in `core_api.h`, same in extended.

Reply:
My point was that since `core_api.h` includes `core_init.h` and we won't  use `core_init.h`  without using `core_api.h` they should be merged